rules/features.bzl

```diff
@@ -0,0 +1,4 @@
+feature_names = struct(
+    # Virtualize means that swift,clang read from llvm's in-memory file system
+    virtualize_frameworks = "apple.virtualize_frameworks",
```

Comment:
If there any reason why we treat it as a feature? Could we simply delete the old way to simplify the code?

Reply:
Yeah it's easier to iterate on code that's behind a feautre. This and we don't want to disrupt the old code path and may roll this out as an a/b test. Probably longer term we'd want to remove the useage of frameworks altogether

Reply:
( that is, no remove the "frameworks" as a feature )